packages/lit-dev-content/samples/examples/react-basics/react.ts

```diff
@@ -0,0 +1,12 @@
+declare global {
+  interface Window {
+    React: any;
+    ReactDOM: any;
+  }
+}
+
+// https://github.com/facebook/react/issues/10021
+const React = window.React;
+const ReactDOM = window.ReactDOM;
+
+export { React, ReactDOM };
```

Comment:
Put `export` in front of `const`s above.

Reply:
done!